test/js-native-api/test_finalizer/test.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+// Flags: --expose-gc
+
+const common = require('../../common');
+const test_finalizer = require(`./build/${common.buildType}/test_finalizer`);
+const assert = require('assert');
+
+(() => {
+  const obj = {};
+  test_finalizer.addFinalizer(obj);
+})();
+
+for (let i = 0; i < 10; ++i) {
+  global.gc();
```

Comment:
Can we make this an async test that uses `gcUntil` like the one below?

Reply:
The point of the test is to show that the "pure" finalizers are run in the current loop "tick". We cannot use `gcUtil` since it is an asynchronous function. I have added a comment.